docs/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml

```diff
@@ -123,31 +126,24 @@
           <para>Install HPCC.</para>
 
           <para>Download the installation package from <ulink
-          url="http://hpccsystems.com/download/free-community-edition">http://hpccsystems.com/download/free-community-edition</ulink>
+          url="http://hpccsystems.com/download/free-community-edition">http://hpccsystems.com/download</ulink>
           and install.</para>
 
           <para>On CentOS/Red Hat:</para>
 
           <para><programlisting> sudo yum install hpccsystems-platform&lt;rpm_file_name&gt;</programlisting>On
           Ubuntu/Debian:</para>
 
-          <programlisting>sudo dpkg -i &lt;deb filename&gt;</programlisting>
+          <programlisting> sudo apt-get install -f  &lt;deb filename&gt;</programlisting>
         </listitem>
```

Comment:
Shouldn't this be ...
'sudo dpkg -i <Deb filename> && sudo apt-get install -f' ?